engine/runtime/src/main/scala/org/enso/compiler/codegen/IrToTruffle.scala

```diff
@@ -21,14 +22,7 @@ import org.enso.compiler.pass.analyse.{
   TailCall
 }
 import org.enso.compiler.pass.optimise.ApplicationSaturation
-import org.enso.compiler.pass.resolve.{
-  ExpressionAnnotations,
-  GenericAnnotations,
-  GlobalNames,
-  MethodDefinitions,
-  Patterns,
-  TypeSignatures
-}
+import org.enso.compiler.pass.resolve._
```

Comment:
In Java I prefer explicit imports rather than wild card ones - they give the code more clarity. I guess Scala deserves the same treatment.